spec/components/progress_tab/diabetes/missed_visits_component_spec.rb

```diff
@@ -72,6 +72,10 @@
     expect(subject).to have_css("div.mb-8px.p-16px.bgc-white.bs-card")
   end
 
+  it "renders the title correctly" do
+    expect(subject).to have_text("Diabetes")
+  end
+
```

Comment:
Added this spec in diabetes as this has to be in, i think i missed this one during DM progress tab